CONTRIBUTING.md

```diff
@@ -1,6 +1,25 @@
 # How to Contribute
 
-## Test Cloud Run Button's Underlying Command Locally
+## Test Cloud Run Button's Underlying Command Locally with Local Go
+
+1. Download Go 1.12.x
+1. Run the tests:
+```
+go test ./cmd/cloudshell_open
+```
+1. Build the command:
+```
+go build -o /tmp/cloudshell_open ./cmd/cloudshell_open
+```
+1. To, test the command you'll need a GCP service account credentials file, then run:
+```
+gcloud components update
+export GOOGLE_APPLICATION_CREDENTIALS=PATH_TO_YOUR_SERVICE_ACCOUNT_KEY_FILE
```

Comment:
you don't need a SA or Key at all, if you just do `gcloud auth application-default login`.